### ISSUE TITLE
apm missing

### ISSUE THREAD

**Issue by plexigras** [atom/apm#711]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

i accidentally uninstalled apm and so i reinstalled atom while i moved all my globaly insalled packages to a local user folder and changed the npm prefix npm config set prefix .local/share/npm.

so i simply did pacman -S atom to reinstall atom but apm is still nowhere to be found

### Versions

1.16.0

**Comment by hantsy** [comment]
When I installed Atom 1.18 under Windows 10, I also can not find APM in the ~/AppData/Local/atom folder.  **APM is missing when I installed the new version**.

I have removed the old atom version, and removed all atom files(~/AppData/Local/atom and ~/.atom) before I installed it.

**Comment by hantsy** [comment]
I found the new APM is located in ~\AppData\Local\atom\app-1.18.0\resources\cli

**Comment by manngo** [comment]
I have the same problem and what looks like a working solution here:

https://discuss.atom.io/t/atom-bin-folder-is-not-always-installed/81279

TLDR:

	%userprofile%\AppData\Local\atom\atom.exe --squirrel-install

will install the folder.
